core/src/main/resources/db/changelog/changesets/2024_10_17_withdrawal_reason.yaml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - changeSet:
+      id: "survey_assign_rename"
+      author: dbush
+      changes:
+        - addColumn:
+            tableName: withdrawn_enrollee
+            columns:
+              - column: { name: reason, type: text }
+        - addColumn:
+            tableName: withdrawn_enrollee
+            columns:
+              - column: { name: note, type: text }
+        - sql:
+            sql: update withdrawn_enrollee set reason = 'TESTING'; # as of now, no enrollees have requested to be withdrawn
+        - addNotNullConstraint:
+            tableName: withdrawn_enrollee
+            columnName: reason
+        - dropColumn:  # closing the loop on the column rename from 2024_09_18_survey_assign_rename.yaml
+            tableName: survey
```

Comment:
nit: I think this one should go in a separate changeset since it's unrelated to the rest of the changes